index.ts

```diff
@@ -20,64 +21,83 @@ app.use(express.json());
 
 app.post("/webhook", async (req, res) => {
   console.log("req receieved");
-  const signature = req.headers["x-hub-signature"];
+  const signature = req.headers["x-hub-signature"] as string;
+  console.log(signature)
```

Comment:
please remove the console.log statements, that do not inform about critical things happening.

Reply:
removed console statements